docs/user/range_time.md

```diff
@@ -0,0 +1,103 @@
+# Range-Time Parameter Plots 
+---
+
+Range-time parameter plots (also known as range-time intensity (RTI) plots) plot a radar measured parameter along range gates versus time. They are the most common way to look at data from a single radar. 
+
+!!! Note
+    Slant ranges are future enhancement we are currently working on. Sorry for inconvenience.
+
+### Basic RTP
+The general syntax for plot_range_time is:
+'plot_range_time(fitacf_data, options)'
+where 'fitacf_data' is the read in data, and the options are several python parameters used to control how the plot looks.
+
+First, make sure Pydarn and matplotlib are imported, then read in the .fitacf file with the data you wish to plot:
+```python
+import pydarn
+import matplotlib.pyplot as plt
+
+fitacf_file = "path/to/fitacf/file"
+sdarn_read = pydarn.SDarnRead(fitacf_file)
+fitacf_data = sdarn_read.read_fitacf()
+
+```
+
+You can choose one of four data products to plot:
+
+| Data product                | String name |
+|-----------------------------|-------------|
+| Line of sight velocity (m/s)| 'v'         |
+| Spectral width (m/s)        | 'w_l'       |
+| Elevation angle (degrees)   | 'elv'       |
+| Power (dB)                  | 'p_l'       |
+
+which is chosen by adding: 
+'parameter=String name' 
+as an option. The default if left blank is 'v'.
+
+To specify which beam to look at, add the option:
+'beam=beam_number"
+
+As an example, taking a look at some 'v' data from the first record of Clyde River radar FIRACF file:
+```python
+fitacf_file = "20190831.C0.cly.fitacf"
+sdarn_read = pydarn.SDarnRead(fitacf_file)
+fitacf_data = sdarn_read.read_fitacf()
+
+pydarn.RTP.plot_range_time(fitacf_data, beam_num=fitacf_data[0]['bmnum'])
+plt.title("Radar {:d}, Beam {:d}".format(fitacf_data[0]['stid'], fitacf_data[0]['bmnum']))  
+
+plt.show()
+```
+which produces:
+
+![](../imgs/rtp_cly1.png)
+
+`fitacf_data[0]['bmnum']` is used to extract the beam number of the first (0th) record from the data dictionary, whilst `fitacf_data[0]['stid']` gives the station id (which is 66 for Clyde River).
+
+Notice that the velocity scale on the right is a bit larger than we need, and also ground scatter isn't shown by default. Showing the dates on the x axis is also a bit redundant, because it's data from a single day. Below, there are some additional parameters you can set to address these and more.
```

Comment:
"ground scatter isn't highlighted by default".  Because it is plotted.

Reply:
Actually grounscatter is not plotted in that example. Velocity colormap has a middle color of "pink" to show the zero/low velocities. Request from @billetd :p 

However, I can see where the confusion it. @billetd can you show a comparison between the two plots showing groundscatter on and off so people don't confuse the pink? Or you probably could just mention groundscatter is grey and explain the pink in the color bar. 

@aburrell is it groundscatter or ground scatter?

Reply:
I think Angeline is correct with this. If I understand right, the groundscatter is 'shown' on the plot, it's just coloured by whatever velocity is calculated for it. If the grounscatter flag is enabled, it just turns these grey, right?

Reply:
```suggestion
Notice that the velocity scale on the right is a bit larger than we need, and also ground scatter coloured grey by default. Showing the dates on the x axis is also a bit redundant, because it's data from a single day. Below, there are some additional parameters you can set to address these and more.
```

Reply:
@billetd is right.  Also, it is usually "groundscatter", though "ground scatter" is... acceptable.